static/app/views/performance/vitalDetail/vitalsComparison.tsx

```diff
@@ -0,0 +1,227 @@
+  const numDecimals = value >= 10_000 ? 1 : 2;
+  const timeInSeconds = value / 1000.0;
+  return (
+    <MetricsCardWrapper>
+      <MetricsTitle>
+        {title} (p75) <StyledQuestionTooltip title={tooltip} size="xs" />
+      </MetricsTitle>
+      <ScoreWrapper>
+        <ScoreContent>{timeInSeconds.toFixed(numDecimals)}s</ScoreContent>
+        <TagWrapper>
+          <StyledTag type={getTagLevel(score)}>{getIndicatorString(score)}</StyledTag>
+        </TagWrapper>
+      </ScoreWrapper>
+    </MetricsCardWrapper>
+  );
+}
+
+function ContentWrapper({
+  organization,
+  vital,
+  children,
+  count,
+}: {
+  children: React.ReactNode;
+  count: number;
+  organization: Organization;
+  vital: WebVital;
+}) {
+  useEffect(() => {
+    trackAdvancedAnalyticsEvent('performance_views.vital_detail.comparsion_viewed', {
+      organization,
+      vital,
+      count,
+    });
+  });
+  return <Container>{children}</Container>;
+}
+
+function VitalsComparison(props: Props) {
+  const {location, vital: _vital, organization} = props;
+  const vitals = Array.isArray(_vital) ? _vital : [_vital];
+  const vital = vitals[0];
+  if (!SUPPORTED_VITALS.includes(vital)) {
+    return null;
+  }
+  return (
+    <VitalsCardDiscoverQuery location={location} vitals={vitals}>
+      {({isLoading, vitalsData}) => {
+        if (isLoading || !vitalsData) {
+          return null;
+        }
+        const {p75} = vitalsData[vital];
+        if (!p75) {
+          return null;
+        }
+        const lookupName = vital === 'measurements.fcp' ? 'FCP' : 'LCP';
+        const sentryStandard = SENTRY_CUSTOMERS[lookupName];
+        const industryStandard = INDUSTRY_STANDARDS[lookupName];
+        const count = vitalsData[vital].total;
+        // only show it if we hit the min number
+        if (count < MIN_VITAL_COUNT_FOR_DISPLAY) {
+          return null;
+        }
+        return (
+          <ContentWrapper {...{organization, vital, count}}>
+            <MetricsCard
+              title={t('Selected Projects')}
+              vital={vital}
+              value={p75}
+              tooltip={tct(
+                "25% of your project's transactions have an [lookupName] greater than this number. Good, Bad, Meh segmentation is based on Google industry standards.",
+                {lookupName}
+              )}
+            />
+            <MetricsCard
+              title={t('Sentry Peers')}
+              vital={vital}
+              value={sentryStandard}
+              tooltip={tct(
+                'Calculated as 50th percentile of top 33% p75 [lookupName] time across all web projects instrumenting their performance with Sentry.',
+                {lookupName}
+              )}
+            />
+            <MetricsCard
+              title={t('Industry Standard')}
+              vital={vital}
+              value={industryStandard}
+              tooltip={tct(
+                "Calculated as a Good [lookupName] based on Google's web.dev standards.",
```

Comment:
```suggestion
                "Calculated as a Good [lookupName] based on Google's industry standards.",
```
for consistency with the first tooltip